npm/nwpolicy_test.go

```diff
@@ -78,14 +78,17 @@ func TestAddNetworkPolicy(t *testing.T) {
 		},
 		Spec: networkingv1.NetworkPolicySpec{
 			Ingress: []networkingv1.NetworkPolicyIngressRule{
-				networkingv1.NetworkPolicyIngressRule{
+				// networkingv1.NetworkPolicyIngressRule
+				{
 					From: []networkingv1.NetworkPolicyPeer{
-						networkingv1.NetworkPolicyPeer{
+						// networkingv1.NetworkPolicyPeer
+						{
 							PodSelector: &metav1.LabelSelector{
 								MatchLabels: map[string]string{"app": "test"},
 							},
 						},
-						networkingv1.NetworkPolicyPeer{
+						// networkingv1.NetworkPolicyPeer
```

Comment:
I gather these were just redundant type declarations. Better remove than comment. Same for all below.